server/models/Category.js

```diff
@@ -17,10 +17,9 @@ const CategorySchema = new Schema({
         default: true,
     },
     icon: {
-        type: String,
-        default :"default-icon"
-    },
-    order:Number
+        type: String
```

Comment:
why remove default?

Reply:
it always save the value of icon "" like this I don't know why so with default icon it still save it as empyt string because of that I delete and I make condition if it's empty set it default from client side

Reply:
just set it to `required` for now (and set the input text field in the admin to required too)